README.md

```diff
@@ -124,6 +185,11 @@ When compatibility with runtime is required (i.e for a standalone build), the re
 
 >  **Note:** This feature has no dependency to and is not based on the Recorder package.
 
+## Unsupported
```

Comment:
Should we have this section?
If so, should we expand it with features that are on the TODO list as well?

Reply:
As per our chat in the Google docs features list, I've added the Unsupported section here!